react-ui-kit/src/Inputs/Text/utils-style.ts

```diff
@@ -1,4 +1,4 @@
-import { ITheme } from '../../palette/variables';
+import type { ITheme } from '../../palette';
 import { Status } from '../common';
 
 export const getIconColor = (errorActive = false, status: Status, theme: ITheme) => {
```

Comment:
```suggestion
export const getIconColor = (errorActive = false, status: Status, theme: ITheme) => {
  if (status === 'error' && !errorActive) return theme.colors.error[60];
  return theme.colors.neutral[50];
```
Либо через тернарник. Даже короче получится:
```suggestion
export const getIconColor = (errorActive = false, status: Status, theme: ITheme) => {
  return status === 'error' && !errorActive ? theme.colors.error[60] : theme.colors.neutral[50];
```